README.org

```diff
@@ -163,9 +163,9 @@ To customize it and customize its icon;
 (add-to-list 'dashboard-items '(agenda) t)
 #+END_SRC
 
-To show agenda for the upcoming seven days set the variable ~show-week-agenda-p~ to ~t~.
+To show agenda for the upcoming seven days set the variable ~dashboard-week-agenda~ to ~t~.
 #+BEGIN_SRC elisp
-(setq show-week-agenda-p t)
+(setq dashboard-week-agenda t)
```

Comment:
May I ask, why we are specifying another variable but we could just use `show-week-agenda-p`? 😕

Reply:
It's a rename of the variable to match conventions:
https://www.gnu.org/software/emacs/manual/html_node/elisp/Coding-Conventions.html
In this case to let all known that `show-week-agenda-p` belongs to `dashboard` so it's `dashboard-week-agenda-p` but the suffix `-p` could be avoided since it's a boolean variable, that's `dashboard-week-agenda`

Reply:
Got it! My first thought is that `show-week-agenda-p` is an external variable that belongs to `org-agenda` or `calendar`. I just realized it belongs to this package itself. Thanks for the confirmation!